Cisco.Otel.Distribution.Tracing/Tracer.cs

```diff
@@ -1,23 +1,35 @@
 ﻿using OpenTelemetry;
 using OpenTelemetry.Trace;
+using Cisco.Opentelemetry.Specifications.Consts;
 
 namespace Cisco.Otel.Distribution.Tracing;
 
 public static class Tracer
 {
     public static TracerProvider Init()
     {
-        return
+        var tracerProvider =
```

Comment:
I love the previous cleaner return :) any reason why setting it into a variable?

Reply:
i wanted to add a log after the function is running, but before the return